modules/rubiconBidAdapter.js

```diff
@@ -535,11 +525,14 @@ function mapSizes(sizes) {
     }, []);
 }
 
-function parsePosition(position) {
-  if (position === 'atf' || position === 'btf') {
-    return position;
-  }
-  return 'unknown';
+/**
+ * Test if bid has mediaType or mediaTypes set for video.
+ * note: 'mediaType' has been deprecated, however support will remain for a transitional period
+ * @param {BidRequest} bidRequest
+ * @returns {boolean}
+ */
+export function hasVideoMediaType(bidRequest) {
+  return bidRequest.mediaType === VIDEO || typeof utils.deepAccess(bidRequest, `mediaTypes.${VIDEO}`) !== 'undefined';
```

Comment:
So `bidRequest.mediaType === VIDEO` is supporting legacy way of declaring video bid.

But in the Validations we return false if mediaTypes.video.context is neither instream or outstream? (Line 115)

Am I missing something or would a legacy video bid where mediaType == 'video' no longer ever work because of this?